db/migrate/20160921184937_add_cancel_reason_detail_to_replication_transfer.rb

```diff
@@ -0,0 +1,5 @@
+class AddCancelReasonDetailToReplicationTransfer < ActiveRecord::Migration
+  def change
+    add_column :replication_transfers, :cancel_reason_detail, :text, null: true
+  end
+end
```

Comment:
I've bumped into some MySQL migration issues where it requires a limit on a text field.  (I haven't tried to test these migrations on MySQL yet.)  Adding an explicit limit might be good, to wrestle some control away from the specific-db defaults.